config/output_schemas/intram_gb_itl1.toml

```diff
@@ -2,14 +2,10 @@
 old_name = "None"
 Deduced_Data_Type = "str"
 
-["Country or region (ITL1)"]
+["Region (ITL1)"]
 old_name = "None"
 Deduced_Data_Type = "str"
 
-["2023"]
+["2022 Total q211"]
 old_name = "None"
```

Comment:
Can we please remove all references to the year, in this one and other places? Just say "Year total q211"